app/frontend/src/components/UI/teammate-list/TeammateListItem.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ReactComponent as Avatar } from 'icons/DefaultAvatar.svg';
+import Icon from '../icon/Icon';
+import { removeTeammateFx } from 'store/teammates';
+
+/**
+ * Interface for teammate component props
+ */
+export interface Props {
+  /**
+   * Teammate id
+   */
+  _id: string
+
+  /**
+   * Teammate name
+   */
+  name: string
+
+  /**
+   * Teammate picture
+   */
+  photo?: string
+
+  /**
+   * True if item is active
+   */
+  isActive?: boolean
+}
+
+/**
+ * Styled teammate component
+ *
+ * @param props - props of the component
+ */
+const StyledTeammateListItem = styled.li<Props>`
+  touch-callout: none;
+  user-select: none;
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  margin-bottom: 7px;
+  width: 100%;
+  height: 38px;
+  border-radius: 12px;
+
+  &:hover {
+    background: var(--color-bg-hover);
+
+    span {
+      display: block;
+    }
+`;
+
+/**
+ * Styled teammate avatar
+ */
+const TeammateAvatar = styled.div`
+  height: 24px;
```

Comment:
maybe it would be better to move the `24` value to the variable

Reply:
Fixed